.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
+ignore = W503
```

Comment:
This is something I find pretty frustrating. Sometimes you just _have_ to break a long line with a multi-condition `if`-statement in it. W503 says you can't break before the binary operator, but W504 says you can't break after the binary operator. Looking at the [PEP8 recommendations](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator), I usually opt to break before the binary operator. Doing so requires telling linters like `flake8` to ignore W503.

Reply:
This hurts my head.